apps/site-landing/src/containers/HelpDocs/styles.module.less

```diff
@@ -0,0 +1,28 @@
+@import "~@thecointech/site-semantic-theme/variables.less";
+
+.containerFaq{
+    width: 50%;
+    max-width: 600px;
+    margin: auto;
+}
+
+#menuFaq{
+    a {
+        color: @theCoinPaletteGreen5;
```

Comment:
we need to change the default styling of links - rather than setting then removing styling like this